cypress/features/regression/portrait.feature

```diff
@@ -0,0 +1,97 @@
+Feature: Portrait default component
+	I want to test Portrait default component
```

Comment:
to large indents

Reply:
on my local it's good. I think that is some git issue